.teamcity/src/main/kotlin/projects/StageProject.kt

```diff
@@ -70,19 +66,7 @@ class StageProject(
             .map { FunctionalTest(model, it.asConfigurationId(model), it.asName(), it.asName(), it, stage = stage, enableTestDistribution = false) }
         topLevelFunctionalTests.forEach(this::buildType)
 
-        val functionalTestProjects = allCoverage
-            .map { testCoverage ->
-                val functionalTestProject = FunctionalTestProject(model, functionalTestBucketProvider, testCoverage, stage)
-                if (stage.functionalTestsDependOnSpecificBuilds) {
```

Comment:
You were able to remove this because it has been basically unused? Since `QuickFeedback` doesn't have any specific builds and it was the only one which used this flag?

Reply:
Right.